lexicons/app/bsky/unspecced/searchPostsSkeleton.json

```diff
@@ -0,0 +1,52 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.searchPostsSkeleton",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Backend Posts search, returning only skeleton",
+      "parameters": {
+        "type": "params",
+        "required": ["q"],
+        "properties": {
+          "q": {
+            "type": "string",
+            "description": "search query string; syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended"
+          },
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 100,
+            "default": 25
+          },
+          "cursor": {
+            "type": "string",
+            "description": "optional pagination mechanism; may not necessarily allow scrolling through entire result set"
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["posts"],
+          "properties": {
+            "cursor": { "type": "string" },
+            "hitsTotal": {
+              "type": "integer",
+              "description": "count of search hits. optional, may be rounded/truncated, and may not be possible to paginate through all hits"
+            },
+            "posts": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "format": "at-uri"
+              }
+            }
```

Comment:
Same goes here, there could be some value in wrapping the uri in an object so that we leave open an upgrade path to add additional/meta info about each result in the future.